cockroachdb/metadata.csv

```diff
@@ -1,261 +1,396 @@
+cockroachdb.distsender.rpc.sent.total,count,,,,[Open Metrics v1 & v2] Number of RPCs sent,0,cockroachdb,
+cockroachdb.exec.error,count,,request,,[Open Metrics v1 & v2] Number of batch KV requests that failed to execute on this node,0,cockroachdb,
+cockroachdb.exec.error.count,count,,request,,[Open Metrics v2] Number of batch KV requests that failed to execute on this node,0,cockroachdb,
+cockroachdb.exec.latency,gauge,,nanosecond,,[Open Metrics v1] Latency in nanoseconds of batch KV requests executed on this node,0,cockroachdb,
+cockroachdb.exec.latency.bucket,gauge,,nanosecond,,[Open Metrics v2] Latency in nanoseconds of batch KV requests executed on this node,0,cockroachdb,
+cockroachdb.exec.success,count,,request,,[Open Metrics v1 & v2] Number of batch KV requests executed successfully on this node,0,cockroachdb,
+cockroachdb.exec.success.count,count,,request,,[Open Metrics v2] Number of batch KV requests executed successfully on this node,0,cockroachdb,
+cockroachdb.gcbytesage,gauge,,second,,[Open Metrics v1 & v2] Cumulative age of non-live data in seconds,0,cockroachdb,
+cockroachdb.gossip.bytes.received,count,,byte,,[Open Metrics v1 & v2] Number of received gossip bytes,0,cockroachdb,
+cockroachdb.gossip.bytes.received.count,count,,byte,,[Open Metrics v2] Number of received gossip bytes,0,cockroachdb,
+cockroachdb.gossip.bytes.sent,count,,byte,,[Open Metrics v1 & v2] Number of sent gossip bytes,0,cockroachdb,
+cockroachdb.gossip.bytes.sent.count,count,,byte,,[Open Metrics v2] Number of sent gossip bytes,0,cockroachdb,
+cockroachdb.gossip.connections.incoming,gauge,,connection,,[Open Metrics v1 & v2] Number of active incoming gossip connections,0,cockroachdb,
+cockroachdb.gossip.connections.outgoing,gauge,,connection,,[Open Metrics v1 & v2] Number of active outgoing gossip connections,0,cockroachdb,
+cockroachdb.gossip.connections.refused,count,,connection,,[Open Metrics v1 & v2] Number of refused incoming gossip connections,0,cockroachdb,
+cockroachdb.gossip.connections.refused.count,count,,connection,,[Open Metrics v2] Number of refused incoming gossip connections,0,cockroachdb,
+cockroachdb.gossip.infos.received,count,,,,[Open Metrics v1 & v2] Number of received gossip Info objects,0,cockroachdb,
+cockroachdb.gossip.infos.received.count,count,,,,[Open Metrics v2] Number of received gossip Info objects,0,cockroachdb,
+cockroachdb.gossip.infos.sent,count,,,,[Open Metrics v1 & v2] Number of sent gossip Info objects,0,cockroachdb,
+cockroachdb.gossip.infos.sent.count,count,,,,[Open Metrics v2] Number of sent gossip Info objects,0,cockroachdb,
+cockroachdb.intentage,gauge,,second,,[Open Metrics v1 & v2] Cumulative age of intents in seconds,0,cockroachdb,
+cockroachdb.intentbytes,gauge,,byte,,[Open Metrics v1 & v2] Number of bytes in intent KV pairs,0,cockroachdb,
+cockroachdb.intentcount,gauge,,key,,[Open Metrics v1 & v2] Count of intent keys,0,cockroachdb,
```

Comment:
Is this correct? The description says count although it is submitted as gauge. V2 fixes the issue with counters being submitted as gauge I believe.